src/scanspec/core.py

```diff
@@ -1,19 +1,25 @@
 from __future__ import annotations
 
+import itertools
 from collections.abc import Callable, Iterable, Iterator, Sequence
 from functools import lru_cache
 from inspect import isclass
-from typing import Any, Generic, Literal, TypeVar
+from typing import (
+    Any,
+    Generic,
+    Literal,
+    TypeVar,
+)
 
 import numpy as np
+import numpy.typing as npt
 from pydantic import BaseModel, ConfigDict, Field, GetCoreSchemaHandler
 from pydantic.dataclasses import is_pydantic_dataclass, rebuild_dataclass
 from pydantic_core import CoreSchema
 from pydantic_core.core_schema import tagged_union_schema
 
 __all__ = [
     "if_instance_do",
-    "Axis",
```

Comment:
This was in `__all__` so the docs didn't complain about a missing ref, please can you check the docs still look ok after this?

Reply:
Since I agree with https://github.com/bluesky/scanspec/pull/143#discussion_r1759070501, I'm putting this back, however I was planning to manually check the docs and the plotting once you were happy with everything else, so will leave this thread open as a placeholder for that.